extensions/amp-story/1.0/test/test-amp-story-grid-layer.js

```diff
@@ -127,6 +127,28 @@ describes.realWin('amp-story-grid-layer', {amp: true}, (env) => {
     ).to.equal(562);
   });
 
+  it('should use the scaling factor to set the size of the layer', async () => {
+    gridLayerEl.setAttribute('aspect-ratio', '1:2');
+    gridLayerEl.setAttribute('scaling-factor', '2');
+    await buildGridLayer();
+
+    storeService.dispatch(Action.SET_PAGE_SIZE, {width: 1000, height: 1000});
+
+    expect(gridLayerEl).to.have.class('i-amphtml-story-grid-template-aspect');
+    expect(
+      parseInt(
+        gridLayerEl.style.getPropertyValue('--i-amphtml-story-layer-width'),
+        10
+      )
+    ).to.equal(1000);
+    expect(
+      parseInt(
+        gridLayerEl.style.getPropertyValue('--i-amphtml-story-layer-height'),
+        10
+      )
+    ).to.equal(2000);
```

Comment:
Plz add one more test to check the default size with no `scaling-factor` attribute

Reply:
Anything extra from the normal `aspect-ratio` tests?